mobile-version.html

```diff
@@ -0,0 +1,124 @@
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job-3">
+                <img src="/images/Snapshoot-Portfolio-3.jpg" alt="job-3 image">
+                <div class="content">
+                    <h3>Tonic</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job-4">
+                <img src="/images/Snapshoot-Portfolio-2.jpg" alt="job-4 image">
+                <div class="content">
+                    <h3>Multi-Post Stories</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="About">
+            <h2>About <br> Myself</h2>
+            <p class ="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+                <div class="contacts">
+                    <ul>
+                        <li><a href="#" class ="link"><i class="fa fa-twitter"></i></a>
+                        <li><a href="#" class ="link"><i class="fa fa-linkedin-square" ></i></a></li>
+                        <li><a href="#" class ="link"><i class="fa fa-github"></i></a></li>
+                        <li><a href="#" class ="link"><i class= "fa fa-hand-peace-o"></i></a></li>
+                        <li><a href="" class="link"><i class="material-icons">feedback</i></a></li>
+                    </ul>
```

Comment:
- [ ] As I can see i think it would be better if you stick with the design icon so please update them to be the same and put the same icons you can import from the Figma design with the same style margin and padding.🙏🏻🙏🏻 
  - your design 
  
![image](https://user-images.githubusercontent.com/58553711/177006057-708c2d96-6ce0-4c31-9ebe-9ac69cbc6b29.png)

  - Figma design
  
![image](https://user-images.githubusercontent.com/58553711/177006000-0e62eb60-1994-4f31-a7e8-5a0e6af7f434.png)

Reply:
I submitted the first project with the same icons so I decided to keep them.
Otherwise, I am not able to find some icons such as M ..